programs/wen_new_standard/src/instructions/mint/royalties/modify.rs

```diff
@@ -71,9 +71,10 @@ impl<'info> ModifyRoyalties<'info> {
 
 pub fn handler(ctx: Context<ModifyRoyalties>, args: UpdateRoyaltiesArgs) -> Result<()> {
     let mint_account = ctx.accounts.mint.to_account_info().clone();
-    let mint_account_data = mint_account.try_borrow_mut_data()?;
+    let mint_account_data = mint_account.try_borrow_data()?;
     let mint_data = StateWithExtensions::<BaseStateMint>::unpack(&mint_account_data)?;
     let metadata = mint_data.get_variable_len_extension::<TokenMetadata>()?;
+    drop(mint_account_data);
```

Comment:
```suggestion
    let mint_account_data = mint_account.try_borrow_data()?;
    let mint_data = StateWithExtensions::<BaseStateMint>::unpack(&mint_account_data)?;
    let metadata = mint_data.get_variable_len_extension::<TokenMetadata>()?;
    drop(mint_account_data);
```

Looks like the block version was reverted `{}`

Reply:
Yeah applying this in the new fixes PR @kespinola